themes/grass/layouts/partials/footer.html

```diff
@@ -28,5 +71,11 @@
 <script>hljs.initHighlightingOnLoad();</script>
 <script src="{{ .Site.BaseURL }}plugins/gallery/photoswipe.js"></script>{{ $script := resources.Get "js/script.js" | minify}}
 <script src="{{ $script.Permalink }}"></script>
+<!-- this is for cookies -->
+<script type="module" src="{{ .Site.BaseURL }}plugins/cookies/cookify_loader.js"></script>
+<script type="text/javascript" data-c-script="analysis">
+    if (window.track) window.track()
+</script>
```

Comment:
guess we could just use what they provide: https://developer.matomo.org/guides/tracking-javascript-guide
alternatively put back this changes https://github.com/OSGeo/grass-website/commit/e48c0a1b8eb97138145968d1d1fc54a3f62f1a60